vpr/src/route/connection_router.h

```diff
@@ -286,8 +291,16 @@ class ConnectionRouter : public ConnectionRouterInterface {
 
     bool only_opin_inter_layer;
 
-    // The path manager for RCV, keeps track of the route tree as a set, also manages the allocation of the heap types
+    // Timing
+    std::chrono::microseconds sssp_total_time{0};
+
+    // Used only by the timing-driven router. Stores the upstream resistance to ground from each RR node, including the
+    // resistance of the node itself (device_ctx.rr_nodes[index].R).
+    vtr::vector<RRNodeId, float> rr_node_R_upstream;
```

Comment:
1. We could move rr_node_R_upstream into rr_node_route_inf as a new data member, as it basically matches the scope and purpose of that structure (data needed per rr_node during routing).
2. This would probably take some refactoring so maybe not worth it, but potentially rr_node_route_inf could be moved into this class.  It is intended to be per rr-node data needed during routing, but we do use some data like congestion of a node for statistics and graphics, so maybe it would get confusing / take too much effort if we moved it into this class.